### ISSUE TITLE
Ingest bavarian LoD2 data

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#160]
## User story
1. As a User
2. I want the Lod2 data 
3. So that I can see a 3d model of a coordingate/address/region

## Depends on
#149 

## Acceptance criteria
* Ingest and access: https://geodaten.bayern.de/opengeodata/OpenDataDetail.html?pn=lod2
* #163 
* #161 
* #162 

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
